workflows/kfdrc_ngs_checkmate_wf.cwl

```diff
@@ -29,8 +31,8 @@ doc: |
       type: ['null', int]
       default: 4000
   ```
-  1) Ram input param optional - use if you plan on batching ~20+ vcfs.
-  2) `input_vcf` is an array of arrays - basically an array of groups of vcfs that you'd like ot see evaluated together.
+  1) Ram in megabytes, input param optional - use if you plan on batching ~20+ vcfs.
```

Comment:
I suppose it's kind of odd we almost always specify RAM in GB for our workflows.

Reply:
Yeah, I didn't want to break bix-ops current workflow by adding in that change...maybe I should and just inform him of the change